service_geo_location/app/controllers/GeoLocationController.scala

```diff
@@ -46,8 +47,19 @@ class GeoLocationController @Inject()(
     val expression = search.getOrElse("")
     geoLocService
       .searchGeoNorway(expression)
-      .map { location =>
-        Ok(Json.toJson(location))
+      .map {
+        case MusitSuccess(locations) => {
```

Comment:
Remove unnecessary `{` and `}` after block starting with `=>`. This applies to all `case` statements below.